packages/tooltip/Tooltip.tsx

```diff
@@ -9,34 +9,43 @@ const TooltipTypes = {
   bottom: 'bottom',
 };
 
-type TooltipTypesType = typeof TooltipTypes;
+type TooltipTypesType = keyof typeof TooltipTypes;
 
 // public
-interface TooltipProps {
-  type?: keyof TooltipTypesType,
+type TooltipProps = {
+  type?: TooltipTypesType,
   label: string,
-}
+};
+
+declare global {
+  namespace JSX {
+    interface IntrinsicElements {
+      'bl-tooltip': TooltipProps & Partial<HTMLElement>,
```

Comment:
So any other component, that has DIV as a root element should have `HTMLDivElement`?
- is this really correct?
- if we don't pass any attributes to the root element